src/uiSelectDirective.js

```diff
@@ -41,6 +41,14 @@ uis.directive('uiSelect',
           }
         }();
 
+        $select.preventPageScroll = function() {
+          if (angular.isDefined(attrs.preventPageScroll)) {
+            return $parse(attrs.preventPageScroll)();
```

Comment:
I used the same $parse function from the others.
it only needs to parse string to boolean.